### PR TITLE
Fix keys locked status, add additional details

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

**Comment by maxtaco** [comment]
👍

File: go/libkb/account.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ type Account struct {
 	skbKeyring   *SKBKeyringFile
 	secSigKey    GenericKey // cached secret signing key
 	secEncKey    GenericKey // cached secret encryption key
-	lksec        *LKSec     // local key security
+	lksec        *LKSec     // local key security (this member not currently used)
```
Comment (patrickxb) [inline go/libkb/account.go]: I kinda would like to keep it for the day when we refactor key unlocking...I added a comment elsewhere in this file about that.  But I understand that it is cruft...

Comment (maxtaco) [inline go/libkb/account.go]: ok

**Comment by patrickxb** [comment]
retest this please

**Pull request merged**